coralproject/talk
# Close comments

Adds functionality for closing comments on a thread and a closed comments message in the admin

## How to test?

- Go to the admin settings
- Create a `closed comments` message
- Go to a thread
- There is a third tab for closing comments
- Close the comments
- Go back to the first tab, the comments tab
- It should display that message
